src/Bigcommerce/Api/Connection.php

```diff
@@ -91,6 +91,9 @@ class Connection
      */
     public function __construct()
     {
+        if (!defined('STDIN')) {
+            define('STDIN', fopen('php://stdin', 'r'));
+        }
         $this->curl = curl_init();
```

Comment:
@PascalZajac I can remove this change to make this PR a proper revert.
I was hoping to resolve the issue(s) reported in https://github.com/bigcommerce/bigcommerce-api-php/issues/139